DYDX-Bots/Frozen-monitor/README.md

```diff
@@ -0,0 +1,37 @@
+# Frozen state monitor
+
+## Description
+
+This bot detects frozen state changes in dYdX perpetual exchange contract.
+
+> Perpetual contract : `0xD54f502e184B6B739d7D27a6410a67dc462D69c8`.
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- DYDX-2-1
+
+  - Fired when `LogFrozen` event is emitted on dYdX perpetual exchange contract.
+  - Severity is always set to "Info".
+  - Type is always set to "Info".
+  - Metadata contains:
+    - `from`: address running the transaction where the event was emitted.
+
+- DYDX-2-2
+  - Fired when `LogUnFrozen` event is emitted on dYdX perpetual exchange contract.
+  - Severity is always set to "Info".
+  - Type is always set to "Info".
+  - Metadata contains:
+    - `from`: address running the transaction where the event was emitted.
+
+## Test Data
+
+The bot behaviour can be verified with the following test transactions generated through our `PoC` contracts (Ropsten):
```

Comment:
I think including the testnet contract addresses, as well as verifying them on etherscan, would be helpful. That way, we can confirm  that the contracts in `PoC/` are indeed the same as the deployed ones.